### PR TITLE
Create BLorient11719

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a record according to the catalogue of St. Strelcyn. I was confused about the composition of the manuscript. 

Strelcyn has recorded different dimensions, layouts, hands and contents. However, changes in dimension, layout, hand or content do not always correlate with each other. The question therefore arises as to whether the manuscript is a composite one and, if so, which parts can be identified.

I was not able to identify the texts in p2 and opened two issues for them: https://github.com/BetaMasaheft/Documentation/issues/2565 and https://github.com/BetaMasaheft/Documentation/issues/2566

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient11719.xml [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+                        </history>
+                    </msPart>
+                    <msPart xml:id="p2">
+                        <msIdentifier/>
+                        <msContents>
+                            <msItem xml:id="p2_i1">
+                                <locus from="3ra" to="28rb"/> 
+                                <title ref="LIT0000000000000000000000000000000000"/>
+                                <note>Amharic commentary-introduction to <ref type="work" corresp="#LIT1716Kidanz">Kidān za-nagh</ref>
+                                    and <ref type="work" corresp="#LIT2444Temher">Tǝmhǝrta ḫǝbuʾat</ref>.</note>
```
Comment (karljonaskarlsson) [inline LondonBritishLibrary/orient/BLorient11719.xml:151]: ḫǝbuʾāt ?

File: LondonBritishLibrary/orient/BLorient11719.xml [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+                                        ጌታ፡ በሕይወተ፡ ሥጋ፡ ሳለ፡ ፫ት፡ ዓመት፡ ከ፫ት፡ ወርኅ፡ ያስተማረው፡ ወንጌል፡ ይባላል፡ ሞቶ፡ ተነስቶ፡ ያስተማረው፡ ኪዳን፡ ይባላል፡ ይሀውም፡ ይታወቅ፡ ዘንድ፡
+                                        ቅዱስ፡ ጳውሎስ፡ በገላትያ፡ ክታቡ፡</incipit>
+                                    <explicit xml:lang="am">ይትእምርተ፡ ኅቡአትና፡ የከዳን፡ መቅድማቸው፡ አንድ፡ ነው። ከእንግዴህ፡ ወዲህ፡ ትእምርተ፡ ኅቡዓት፡ ቢሉ፡ ወይትእምርተ፡
+                                        ኅቡዓት፡ ኪዳን፡ ቢሉ፡ ወይኪዳን፡ ይሔድዋል። ትእምርተ፡ ኅቡዓትም፡ ኪዳንም፡ የጌታ፡ ቃል፡ ስለ <pb n="9ra"/>ሆኑ፡ በየፈንታ፡ ይነገራሉ፡ እንጅ፡ እንድ፡
+                                        ጊዜ፡ አይነግሩም። ነገር፡ ግን፡ ካህን፡ ትእምርተ፡ ኅቡዓት፡ ሳይተረጐም፡ መቅደሱን፡ አይከፍተው፡ ካለ፡ ትእምርተ፡ ኅቡዓት፡ ይቅድም።</explicit>
+                                    <textLang xml:lang="am"/>
+                                </msItem>
+                                <msItem xml:id="p2_i1.2">
+                                    <locus from="9ra"/> 
+                                    <title ref="LIT0000000000000000000000000000000000">Beginning of the commentary on the  tǝmhǝrta ḫǝbuʾat</title>
```
Comment (karljonaskarlsson) [inline LondonBritishLibrary/orient/BLorient11719.xml:166]: ḫǝbuʾāt ?

File: LondonBritishLibrary/orient/BLorient11719.xml [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+                                    <note>Possibly more than one hand.</note>
+                                </handNote>
+                            </handDesc> 
+                        </physDesc>    
+                        <history>
+                            <origin>
+                                <origDate notBefore="1700" notAfter="1799">18th century</origDate>    
+                            </origin>
+                            <provenance>The owner was <persName ref="PRS14491WaldaMaryam">Walda Māryām</persName>, the scribe or 
+                                one of the scribes was <persName ref="PRS14492WaldaGabreel"><roleName type="title">ʾabbā</roleName>
```
Comment (karljonaskarlsson) [inline LondonBritishLibrary/orient/BLorient11719.xml:259]: ʾAbbā ?

File: LondonBritishLibrary/orient/BLorient11719.xml [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+                                    <explicit xml:lang="am">ይትእምርተ፡ ኅቡአትና፡ የከዳን፡ መቅድማቸው፡ አንድ፡ ነው። ከእንግዴህ፡ ወዲህ፡ ትእምርተ፡ ኅቡዓት፡ ቢሉ፡ ወይትእምርተ፡
+                                        ኅቡዓት፡ ኪዳን፡ ቢሉ፡ ወይኪዳን፡ ይሔድዋል። ትእምርተ፡ ኅቡዓትም፡ ኪዳንም፡ የጌታ፡ ቃል፡ ስለ <pb n="9ra"/>ሆኑ፡ በየፈንታ፡ ይነገራሉ፡ እንጅ፡ እንድ፡
+                                        ጊዜ፡ አይነግሩም። ነገር፡ ግን፡ ካህን፡ ትእምርተ፡ ኅቡዓት፡ ሳይተረጐም፡ መቅደሱን፡ አይከፍተው፡ ካለ፡ ትእምርተ፡ ኅቡዓት፡ ይቅድም።</explicit>
+                                    <textLang xml:lang="am"/>
+                                </msItem>
+                                <msItem xml:id="p2_i1.2">
+                                    <locus from="9ra"/> 
+                                    <title ref="LIT0000000000000000000000000000000000">Beginning of the commentary on the  tǝmhǝrta ḫǝbuʾat</title>
+                                    <note>On <locus from="14v" to="27r"/> the rubrics are missing.</note>
+                                    <incipit xml:lang="am">በእንተ፡ ትእምርተ፡ ኅቡአት። ኅቡዓትን፡ የመማር፡ ኅቡዓትን፡ የማስተማር፡ ነገር፡ እንዲህ፡ ነው፡ ብሎ፡ አፍዝዞ፡ ተወው። አንድ፡
```
Comment (karljonaskarlsson) [inline LondonBritishLibrary/orient/BLorient11719.xml:168]: Maybe a "sic" for Tǝʾmǝrta for expected Tǝmhǝrta ?

**Review by karljonaskarlsson — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient11719.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient11719.xml:259]: I thought, that titles are regularly small, but probably this rule applies only for titles in PRS records.

**Review by karljonaskarlsson — CHANGES_REQUESTED** [review]

**Review by karljonaskarlsson — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**